india_compliance/gst_india/api_classes/nic/e_invoice.py

```
@@ -115,3 +121,73 @@
+        if not success_value:
+            self.handle_server_error(response_json)
+
+        if not success_value and not self.is_ignored_error(response_json):
+            frappe.throw(
+                response_json.get("ErrorDetails", [{}])[0].get("ErrorMessage")
+                # Fallback to response body if message is not present
+                or frappe.as_json(response_json, indent=4),
+                title=_("API Request Failed"),
+            )
+
+    def is_ignored_error(self, response):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)